File: BinhDuongHackathon/src/app/shared/su-kien-phan-anh.ts
import { UrlData } from "app/shared/url-data";

export class SuKienPhanAnh {
  public kinhDo: number;
  public viDo: number;
  public thoiGian: string;
  public thoiGianNgan: string;
  public loaiPhanAnh: string;
  public moTa: string;
  public tinhTrang: string;
  public stt: number;
  public dsUrlData: UrlData[];
  public viTri: string;

  constructor(
    kinhDo: number,
    viDo: number,
    thoiGian: string,
    thoiGianNgan: string,
    loaiPhanAnh: string,
    moTa: string,
    tinhTrang: string,
    dsUrlData: UrlData[],
    viTri: string
  ) {
    this.kinhDo = kinhDo;
    this.viDo = viDo;
    this.thoiGian = thoiGian;
    this.thoiGianNgan = thoiGianNgan;
    this.loaiPhanAnh = loaiPhanAnh;
    this.moTa = moTa;
    this.tinhTrang = tinhTrang;
    this.dsUrlData = dsUrlData;
    this.viTri = viTri;
  }
}

File: Android/Binh Duong Emergency/app/src/main/java/model/SuKienPhanAnh.java
package model;

import java.util.List;

public class SuKienPhanAnh {
    public String thoiGian;
    public String thoiGianNgan;
    public double kinhDo;
    public double viDo;
    public String moTa;
    public String loaiPhanAnh;
    List<URLDownload> urlDownloadList;
    public String tinhTrang;
    public String viTri;

    public SuKienPhanAnh(String thoiGian, String thoiGianNgan, double kinhDo, double viDo, String moTa, String loaiPhanAnh, List<URLDownload> urlDownloadList,String viTri,String tinhTrang) {
        this.thoiGian = thoiGian;
        this.thoiGianNgan = thoiGianNgan;
        this.loaiPhanAnh = loaiPhanAnh;
        this.kinhDo = kinhDo;
        this.viDo = viDo;
        this.moTa = moTa;
        this.urlDownloadList = urlDownloadList;
        this.tinhTrang=tinhTrang;
        this.viTri=viTri;
    }
}